app/s3-provider/s3Provider.js

```diff
@@ -0,0 +1,27 @@
+class S3Provider {
+    constructor(s3Client, loggerProvider) {
+        this.client = s3Client;
+        this.logger = loggerProvider.create(S3Provider.name);
+    }
+
+    async putObject(Key, Body) {
+        try {
+            await this.client.putObject({ Bucket: 'pastebin', Key, Body });
+        } catch (error) {
+            this.logger.error(error);
+        }
+    }
+
+    async getObject(Key) {
+        // not really sure if that is a good approach but it works ( .Body.transformToString(); )
```

Comment:
is this approach ok?

Reply:
It is not the best approach, but if it works for now, let it be.
We may got problems at the edge cases, but would manage it later.